internal/tiltfile/local_resource.go

```diff
@@ -37,12 +41,22 @@ func (s *tiltfileState) localResource(thread *starlark.Thread, fn *starlark.Buil
 
 	deps := value.NewLocalPathListUnpacker(thread)
 
-	var resourceDepsVal starlark.Sequence
+	var resourceDepsVal, tagsVal starlark.Sequence
 	var ignoresVal starlark.Value
 	var allowParallel bool
 	var links links.LinkList
 	autoInit := true
 
+	var isTest bool
+	if fn.Name() == testN {
+		// If we're initializing a test, by default parallelism is on
+		allowParallel = true
+
+		isTest = true
+
+		// TODO: implement timeout
```

Comment:
I'm in favor of moving this TODO outside of the `if` statement and allowing timeouts on local_resource cmds, if it doesn't make this PR annoying. I was actually wishing I had this last week.

Reply:
ha yeah I was talking to L about this: "args like `tags` and `timeout` are only relevant to `test()`... though honestly it would be great if you could set them on `local_resource()` as well!"